app/views/admin/settings/forms/_images.html.erb

```diff
@@ -33,36 +33,7 @@
           <img alt="favicon" class="preview" src="<%= Images::Optimizer.call(Settings::General.favicon_url, width: 32) %>" width="32" height="32" loading="lazy" />
         </div>
 
-        <div class="crayons-field">
-          <%= admin_config_label :logo_png %>
-          <%= admin_config_description Constants::Settings::General::DETAILS[:logo_png][:description] %>
-          <%= f.text_field :logo_png,
-                           class: "crayons-textfield",
-                           value: Settings::General.logo_png,
-                           placeholder: Constants::Settings::General::DETAILS[:logo_png][:placeholder] %>
-          <% if Settings::General.logo_png.present? %>
-            <img src="<%= Images::Optimizer.call(Settings::General.logo_png, width: 128) %>" width="128" height="128" loading="lazy" />
-          <% end %>
-        </div>
-
-        <% if FeatureFlag.enabled?(:creator_onboarding) %>
-          <%= render partial: "admin/shared/logo_upload", locals: { f: f, allowed_types: logo_allowed_types, max_file_size: logo_max_file_size, logo: Settings::General.resized_logo } %>
-        <% else %>
-          <div class="crayons-field">
-            <%= admin_config_label :logo_svg %>
-            <%= admin_config_description Constants::Settings::General::DETAILS[:logo_svg][:description] %>
-            <%= f.text_area :logo_svg,
-                            class: "crayons-textfield",
-                            value: Settings::General.logo_svg,
-                            rows: 6,
-                            placeholder: Constants::Settings::General::DETAILS[:logo_svg][:placeholder] %>
-            <% if Settings::General.logo_svg.present? %>
-              <div class="site-logo">
-                <%= Settings::General.logo_svg.html_safe %>
-              </div>
-            <% end %>
-          </div>
-        <% end %>
```

Comment:
Quick question: 
Now that we are no longer excepting `logo_svg`s, is it alright for me to create a follow-up PR that strips the remaining code for `logo_svg`s from the codebase or should I wait until we work on the unified logo RFC that overhauls this anyway?

When I search the codebase, it only appears like there are 9 results, spanning 5 files for `logo_svg`, so a follow-up PR should be relatively quick and straight forward. Thoughts?

cc @Ridhwana @nickytonline

Reply:
> is it alright for me to create a follow-up PR that strips the remaining code for logo_svgs from the codebase

I think so, yes. But let's merge it only after launch because if something goes wrong with the logos, `logo_svg` can be our fallback.

Reply:
Sounds perfect, @Ridhwana! I'll add an issue to our board so that post-launch, we can tackle this if everything goes well. 😊